styleguide/components/components.css

```diff
@@ -0,0 +1,263 @@
+@charset "UTF-8";
```

Comment:
Could you remove this file? I have a feeling these .css files got generated when trying to figure out the SASS compilation